daily-challenges/2024-09/28/khanh0509_28.spec.ts

```diff
@@ -0,0 +1,19 @@
+// javascript
+function isPrime(input) {
+  if (input == 1) return 0;
```

Comment:
Please using !== and === for exact match. The same for other places.
```suggestion
  if (input === 1) return 0;
```